packages/suite/src/utils/suite/date.ts

```diff
@@ -84,14 +85,18 @@ export const calcTicksFromData = (data: { time: number }[]) => {
 };
 
 /**
- * Returns Blockbook-safe current unix timestamp (current time - 3 mins).
+ * Returns Blockbook-safe current unix timestamp (time - 3 mins).
```

Comment:
nit: Please correct me if I got the semantics wrong. I think this function could be documented as this
```suggestion
 * Returns the given timestamp coerced to at most 3 minutes ago.
 * If no timestamp is given a timestamp of 3 minutes ago is returned.
```

It would be nice to briefly explain what does "blockbook safe" mean. I don't know honestly. .)

Reply:
The new comment is perfect. Thanks!